src/integrationtests/java/com/aws/greengrass/integrationtests/deviceauth/EvaluateClientDeviceActionsWithPolicyVariablesTest.java

```diff
@@ -79,6 +80,9 @@ void beforeEach(ExtensionContext context) throws Exception {
 
         // create certificate that client devices can use
         setClientDeviceCertificatePem();
+
+        // clear static wildcard map to ensure tests do not interfere with each other
+        PermissionEvaluationUtils.AUTHZ_WILDCARD_MAP.clear();
```

Comment:
this scares me a bit--as you ran into, static state has its own class of footguns. is there a way we can make this not static? i think you were mentioning it was to make it work w/ policy variable changes, so may need to adjust that first

Reply:
would you mind setting up a PR for `policy-variable-support` branch?

Reply:
Right the issue here was that the `PermissionEvaluationUtils` class uses static methods, so the map needed to be static so it could be referenced within the methods. I could look into making the methods non-static so we can avoid this altogether

Reply:
yep lets do that

Reply:
PR for the `policy-variable-support` branch. This includes the changes to make `PermissionEvaluationUtils` use non static methods. https://github.com/aws-greengrass/aws-greengrass-client-device-auth/pull/415